tests/swarm-test.js

```diff
@@ -25,164 +28,89 @@ process.on('uncaughtException', function (err) {
 })
 
 experiment('Without a Stream Muxer', function () {
-  experiment('tcp', function () {
+  experiment('and one swarm over tcp', function () {
```

Comment:
Ah got it, it was so that you could have the 'beforeEach' and 'afterEach'

Nice! :)